--- a/mbedtls/library/aesni.c	2020-04-21 16:42:08.646674659 +0200
+++ b.mbedtls/library/aesni.c	2020-04-21 16:43:26.000000000 +0200
@@ -53,20 +53,7 @@
  */
 int mbedtls_aesni_has_support( unsigned int what )
 {
-    static int done = 0;
-    static unsigned int c = 0;
-
-    if( ! done )
-    {
-        asm( "movl  $1, %%eax   \n\t"
-             "cpuid             \n\t"
-             : "=c" (c)
-             :
-             : "eax", "ebx", "edx" );
-        done = 1;
-    }
-
-    return( ( c & what ) != 0 );
+    return 1;
 }
 
 /*
